fun f() {
    try {
    }
    catch (<caret>e: Exception) {
        println(e)
    }
}

// NAME: e1